### DISCUSSION TITLE
Not able to see the edit option for index.md file

### DISCUSSION THREAD

**Discussion by SRE-Avinash** [skills/.github#676] [Communicate using Markdown]
I dont see the Edit option for index.md file
![image](https://github.com/user-attachments/assets/abc16c41-7294-4ca9-a49e-edcc037ca5a7)
as mention in the step **Step 6**
![image](https://github.com/user-attachments/assets/241fb770-6d57-48e0-8719-469a08dde019)

**Comment by aflacake** [comment]
Yes, I also had a similar problem. Issue: [problem step 3](https://github.com/skills/communicate-using-markdown/issues/251).
